Runtimes/Core/Readme.md

```diff
@@ -0,0 +1,92 @@
+# SwiftCore
+
+SwiftCore contains core libraries that sit below the platform overlays.
+These include the standard library and associated runtimes, SwiftOnoneSupport,
+and the concurrency runtimes.
+
+## Build Instructions
+
+> [!IMPORTANT]
+> The standard library requires that it is built with a Swift compiler that is
+> at least as new as the standard library sources. You will likely need to
+> build the compiler as you would normally.
+> In these instructions, `<swiftc>` is the path to your just-built Swift
+> compiler.
+
+First create a build directory in the `Core/` directory and navigate to it.
+
+```sh
+mkdir build && cd build
+```
+
+Configure a simple default build. This results in a static build of the standard
+library.
+
+```sh
+cmake -G 'Ninja' -DCMAKE_Swift_COMPILER=<swiftc> ../`
+ninja
+```
+
+To build a dynamic library, pass `-DBUILD_SHARED_LIBS=YES` to CMake.
+
+```sh
+cmake . -DBUILD_SHARED_LIBS=YES
+ninja
+```
+
+To install the standard library to a staging directory, set the `DESTDIR`
+environment variable and run `ninja install`
+
+```sh
+DESTDIR=SwiftCore-Staging ninja install
```

Comment:
I don't think that this is portable. The environment variable handling is not identical on all platforms. If we want to do this, we should encourage the use of `cmake -E env DESTDIR=... ...`